.github/workflows/ci.yml

```diff
@@ -5,10 +5,10 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-18.04
-    container: devkitpro/devkitarm:latest
+    container: devkitpro/devkitarm
 
     steps:
-      - uses: actions/checkout@v1
```

Comment:
Leave this at checkout v1 - this will make things easier for us. See @aspargas2's comment.